lib/tictactoe/messages.rb

```diff
@@ -0,0 +1,24 @@
+ALL_MESSAGES = {
```

Comment:
Better to be scoped to a module or class. `ALL_MESSAGES` inside the body of `Tictactoe` module or `Tictactoe::Messages` class.